lld/wasm/SyntheticSections.cpp

```diff
@@ -133,6 +134,14 @@ void DylinkSection::writeBody() {
 
     sub.writeTo(os);
   }
+  if (!ctx.arg.rpath.empty()) {
```

Comment:
Looks like we use an empty line between subsections above.. maybe do that same here?